eks/ng/role.go

```diff
@@ -238,8 +257,14 @@ func (ts *tester) createRole() error {
 	if ts.cfg.EKSConfig.AddOnNodeGroups.RoleName == "" {
 		return errors.New("cannot create a cluster role with an empty AddOnNodeGroups.RoleName")
 	}
-
-	if err := ioutil.WriteFile(ts.cfg.EKSConfig.AddOnNodeGroups.RoleCFNStackYAMLFilePath, []byte(TemplateRole), 0400); err != nil {
+	tr := templateRole{}
+	tr.ASGPolicy = asgPolicyData
```

Comment:
Only set this when autoscaler is enabled?

Reply:
I think this should be ok ? Because by default these policies will be included to the role if the node-groups (or) clusters are created using "eksctl".

Reply:
ack